src/bacnet/basic/object/netport.c

```diff
@@ -36,6 +36,7 @@
 #include <stddef.h>
 #include <stdlib.h>
 #include <string.h>
+#include "bacnet/config.h"
```

Comment:
bacnet/bacdef.h already includes bacnet/config.h file as a basis, so adding bacnet/config.h to the start of these library is no longer needed (but bacdef.h is needed).